distributed/http/scheduler/api.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import json
+
+from distributed.http.utils import RequestHandler
+
+
+class APIHandler(RequestHandler):
+    def get(self):
+        self.write("API V1")
+        self.set_header("Content-Type", "text/plain")
+
+
+class RetireWorkersHandler(RequestHandler):
+    async def post(self):
+        self.set_header("Content-Type", "text/json")
+        scheduler = self.server
+        try:
+            params = json.loads(self.request.body)
+            workers_info = await scheduler.retire_workers(**params)
+            self.write(json.dumps(workers_info))
+        except Exception as e:
+            self.set_status(400, str(e))
+            self.write(json.dumps({"Error": "Bad request"}))
+
+
+class WorkersToCloseHandler(RequestHandler):
+    async def post(self):
```

Comment:
Why is this a POST? It doesn't modify any resources, does it?

Reply:
This closes workers by name, which is a modification.

Reply:
It only provides the worker to be closed but doesn't close any of them

Reply:
Oh yeah I see. I misunderstood what that function was doing.

@Matt711 I don't think we actually need to expose this method at all. `retire_workers` does everything we need.

Reply:
Since we're planning on using this API in the scheduler, don't we need to know which workers we close before closing them? Because we're going to have to close the Kubernetes pods? @jacobtomlinson

Reply:
It's fine, we should be able to use `retire_workers` which reports the names of the workers it retires. Then we can just clean up the completed pods after they exit.